d2/src/main/java/com/linkedin/d2/balancer/dualread/DualReadLoadBalancerMonitor.java

```diff
@@ -72,7 +71,6 @@ public void reportData(String propertyName, T property, String propertyVersion,
     Cache<String, CacheEntry<T>> cacheToAdd = fromNewLb ? _newLbPropertyCache : _oldLbPropertyCache;
     CacheEntry<T> existingEntry = cacheToAdd.getIfPresent(propertyName);
     String propertyClassName = property.getClass().getSimpleName();
-    boolean isUriProp = property instanceof UriProperties;
```

Comment:
could add a comment to this class and note DualReadLoadBalancerMonitor should never be extended for matching URI properties to make sure it won't be mis-used for URI at all. Otherwise, we could end up having log volume again with the URI logs.

Reply:
Sure, can do. I also made it guaranteed at compile time.